postr/twitter_postr.py

```diff
@@ -0,0 +1,61 @@
+# YouTube Video: https://www.youtube.com/watch?v=wlnx-7cm4Gg
+from tweepy import OAuthHandler
+from tweepy import Stream
+from tweepy.streaming import StreamListener
+
+# Variables that contain the user credentials to access the twitter API.
+ACCESS_TOKEN: str = '1034473360628039680-OflwYTsKwd6xh1nzcRWje6veoHsBDY'
```

Comment:
don't commit any of these keys to version control or any other key. They should be only stored inside the config.ini file.

Reply:
Will do.